deepchem/models/torch_models/chemberta.py

```diff
@@ -0,0 +1,174 @@
+"""
+Huggingface/transformers RoBERTa model for sequence-based property prediction.
+"""
+
+from numpy.lib.npyio import save
+import torch.nn as nn
+import torch.nn.functional as F
+from torch.utils.data import random_split
+import os
+
+class ChemBERTaMaskedLM(nn.Module):
+    # defaults here match HF model: DeepChem/SmilesTokenizer_PubChem_1M.
```

Comment:
It looks like you're using 4-space spacing. Make sure to run `yapf` on these files and convert to DeepChem 2-space spacing

Reply:
yup! right now i'm just working on getting all the components to fit better with the deepchem API + pass the unit test I'm using - still have to fix formatting + commenting :)